bigquery/snippets/snippet_test.go

```diff
@@ -298,14 +247,22 @@ func TestViews(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+
 	srcDatasetID := uniqueBQName("golang_example_view_source")
-	if err := createDataset(client, srcDatasetID); err != nil {
-		t.Errorf("createDataset(%q): %v", srcDatasetID, err)
+	if err := client.Dataset(srcDatasetID).Create(ctx,
```

Comment:
For multi-line function calls, I generally like to have them outside the `if` statement. It can be a little hard to find the condition being checked after the declaration when it's inline.

Reply:
This is in the corpus of tests I still need to break up.  Will revisit that in subsequent PRs.